Content.Client/Lobby/UI/LobbyCharacterPreviewPanel.cs

```diff
@@ -58,7 +55,11 @@ public sealed class LobbyCharacterPreviewPanel : Control
                 Orientation = LayoutOrientation.Vertical
             };
 
-            vBox.AddChild(header);
+            var vSpacer = new BoxContainer
+            {
+                Orientation = LayoutOrientation.Vertical,
+                MinHeight = 5
+            };
```

Comment:
use the type you created for this

Reply:
I missed this one, good catch. I'm pushing a fix